README.md

```diff
@@ -1,159 +1,45 @@
-<img src=".erb/img/erb-banner.svg" width="100%" />
+# Notes App
 
-<br>
+This project is a notes app built using React and Electron. It allows users to create, organize, and search through their notes efficiently. The app features a simple, user-friendly interface designed for productivity and note management.
 
-<p>
-  Electron React Boilerplate uses <a href="https://electron.atom.io/">Electron</a>, <a href="https://facebook.github.io/react/">React</a>, <a href="https://github.com/reactjs/react-router">React Router</a>, <a href="https://webpack.js.org/">Webpack</a> and <a href="https://www.npmjs.com/package/react-refresh">React Fast Refresh</a>.
-</p>
+## Features
 
-<br>
+- **Markdown-based Note-Taking**: Create, edit, and save notes using markdown.
+- **File Organization**: Organize notes into folders and subfolders.
+- **Search Functionality**: Quickly search through notes.
```

Comment:
Feels like too much promise. I'm not sure we will take it up. Can u make this more generic and open. When we add specific features we can update the description to be more verbose.

Reply:
I have updated the readme file, pls check once.